spec/requests/api/stateless/transactions_spec.rb

```diff
@@ -6,10 +6,31 @@
 
   let!(:member) { create(:member, email: 'test@example.com') }
   let!(:customer) { create(:payment_braintree_customer, member: member) }
-  let!(:subscription) { create(:payment_braintree_subscription, customer: customer) }
-  let!(:subscription_transaction) { create(:payment_braintree_transaction, subscription: subscription, customer: customer ) }
-  let!(:standalone_transaction_a) { create(:payment_braintree_transaction, customer: customer ) }
-  let!(:standalone_transaction_b) { create(:payment_braintree_transaction, customer: customer ) }
+  let!(:payment_method) { create(:braintree_payment_method,
+    customer: customer,
+    id: 1432,
+    instrument_type: 'credit card',
+    token: '2ewruo4i5o3',
+    last_4: '2454',
+    email: customer.email,
+    card_type: 'Mastercard')}
+  let!(:subscription) { create(:payment_braintree_subscription,
+    customer: customer,
+    payment_method: payment_method) }
+  let!(:subscription_transaction) { create(:payment_braintree_transaction,
```

Comment:
Absolutely. That had been my intention but it looks like I had forgotten to do that. Thanks for the catch!